build/loc.proj

```diff
@@ -73,6 +72,16 @@
     <Copy SourceFiles="@(EulaFiles->'%(Identity)')" DestinationFiles="@(EulaFiles->'%(DestinationDir)')"/>
   </Target>
 
+  <Target Name="MoveLcgFilesToArtifacts" AfterTargets="Localize" Condition="'$(BuildRTM)' != 'true'">
+    <ItemGroup>
+      <_MoveLcgFiles Include="@(LocalizedToolFiles)" Condition="'%(LocalizedToolFiles.Extension)'== '.lcg'">
+        <DestinationDir>$(ArtifactsDirectory)localize\PLOC\net\%(LocalizedToolFiles.Filename)%(LocalizedToolFiles.Extension)</DestinationDir>
```

Comment:
@(LocalizedToolFiles) is from microbuild target "localize", lcg files are already in this structure Project-Name\15.0\bin\Debug\TargetFramework\PLOC, but in wsr-tc, all lcg files are in same directory, there is no project structure, do we still need to follow that pattern?